central/processlisteningonport/datastore/datastore_impl_test.go

```diff
@@ -2286,13 +2479,13 @@ func (suite *PLOPDataStoreTestSuite) TestRemoveOrphanedPLOPs() {
 
 			err = suite.store.UpsertMany(suite.hasWriteCtx, c.initialPlops)
 			suite.NoError(err)
-			plopCount, err := suite.store.Count(suite.hasReadCtx, search.EmptyQuery())
+			plopCount, err := suite.store.Count(suite.hasAllCtx, search.EmptyQuery())
```

Comment:
Would `suite.hasReadCtx` be enough to count the items in the PLOP store ?

Reply:
Yes. Reverted.